src/scss/bootstrap-lifelines-webshop.scss

```diff
@@ -20,7 +20,7 @@ nav.navbar {
   margin-top: 0.5rem;
 }
 
-h3 {
```

Comment:
what arcane trickery was this

Reply:
Just plain evil, i never want to know who does stuff like that. :sweat_smile: So, don't tell me, better yet, don't search for someone to blame, let's just forget it ever happend.